Source/AviationWeather/Source/metar.cpp

```diff
@@ -639,17 +622,28 @@ cloud_layer metar_info::ceiling() const
 
 flight_category metar_info::flight_category() const
 {
+    if (!visibility_group)
+    {
+        throw aw_exception("Missing visibility group");
+    }
+
     auto ceiling = metar_info::ceiling();
```

Comment:
We should also check if `sky_condition_group` is empty when calling `ceiling()`.  Returning a new `cloud_layer` with `layer_height` of `UINT32_MAX` would be potentially erroneous if sky cover is missing entirely -- this could result in VFR being reported when a field is potentially LIFR.

This relates to issue #34 (unknown flight category), so perhaps this should be addressed separately.